core/station/impl/src/services/system.rs

```diff
@@ -734,39 +687,22 @@ mod install_canister_handlers {
     /// Starts the fund manager service setting it up to monitor the upgrader canister cycles and top it up if needed.
     pub fn monitor_upgrader_cycles(
         upgrader_id: Principal,
-        cycle_obtain_strategy: CycleObtainStrategy,
+        cycle_obtain_strategy: &CycleObtainStrategy,
     ) {
-        print(format!(
-            "Starting fund manager to monitor self {} and upgrader canister {} cycles",
-            id(),
-            upgrader_id.to_text()
-        ));
-
-        FUND_MANAGER.with(|fund_manager| {
-            let mut fund_manager = fund_manager.borrow_mut();
-
-            let mut fund_manager_options = FundManagerOptions::new()
-                .with_interval_secs(24 * 60 * 60) // daily
-                .with_strategy(FundStrategy::BelowEstimatedRuntime(
-                    EstimatedRuntime::new()
-                        .with_min_runtime_secs(14 * 24 * 60 * 60) // 14 days
-                        .with_fund_runtime_secs(30 * 24 * 60 * 60) // 30 days
-                        .with_max_runtime_cycles_fund(1_000_000_000_000)
-                        .with_fallback_min_cycles(125_000_000_000)
-                        .with_fallback_fund_cycles(250_000_000_000),
-                ));
-
-            fund_manager_options = fund_manager_options.with_obtain_cycles_options(
-                SYSTEM_SERVICE.get_obtain_cycle_config(&cycle_obtain_strategy),
-            );
-
-            fund_manager.with_options(fund_manager_options);
+        let fund_strategy = FundStrategy::BelowEstimatedRuntime(
+            EstimatedRuntime::new()
+                .with_min_runtime_secs(14 * 24 * 60 * 60) // 14 days
+                .with_fund_runtime_secs(30 * 24 * 60 * 60) // 30 days
+                .with_max_runtime_cycles_fund(1_000_000_000_000)
+                .with_fallback_min_cycles(125_000_000_000)
+                .with_fallback_fund_cycles(250_000_000_000),
```

Comment:
we need to account for the cycles reservation here to at least make sure the monitored canister has enough cycles to do an upgrade:

```suggestion
                .with_fallback_min_cycles(400_000_000_000)
                .with_fallback_fund_cycles(200_000_000_000),
```

this change would make sure that the monitored canister has at least 300B cycles in an application subnet to perform a install operation, as well as there is `25%` surplus to account for more consumption